bin/fin

```diff
@@ -1295,7 +1308,7 @@ show_help_hosts ()
 	printh "fin hosts [command]" "" "yellow"
 	echo
 	printh "Uses $DOCKSAL_DEFAULT_IP for IP when VirtualBox is used."
-	printh "Uses 127.0.0.1 for IP when DOCKER_NATIVE is set."
+	printh "Uses $DOCKSAL_D4MAC_IP for IP when DOCKER_NATIVE is set."
```

Comment:
Why `DOCKSAL_D4MAC_IP`? If there's no difference for Native App then there should not be 2 different vars for mac (`DOCKSAL_D4MAC_IP`) and win (`DOCKSAL_D4WIN_IP`) or we should just hardcode `127.0.0.1` here. 

If there is a difference, then here should be an `if` clause.